Module4_Labs/Lab6_Sudoku_Solver/Cards/712.md

```diff
@@ -64,40 +77,10 @@ def testPossible(grid,r,c,n,depth,finish):
 		return False
 ```
 
+Step 8: Check if finish evaluates to True.
 On the condition that `complete()` returns True in our if-statement, we will check if `finish` evaluates to True also: 
 
 ```python
 	if complete(duplicate,depth):
 		if finish:
 ```
-
-
```

Comment:
Is there a reason why all of this was deleted? The student should be shown the whole solution together in the easy card :)

Reply:
Hey Sarah, I divided it into two parts, step 5 and 6